repose-aggregator/tests/performance-tests/roles/gatling/files/gatling.conf

```diff
@@ -0,0 +1,132 @@
+#########################
+# Gatling Configuration #
```

Comment:
Oh, I see. That makes sense. So we will still be copying the file, just not including all of the defaults. Totally legit. I'll do that.

Last night my brain was stuck on trying to modify the installed configuration instead, which I wasn't a fan of.